### ISSUE TITLE
Decide on CICD architecture

### ISSUE THREAD

**Issue by jandegen** [amosproj/amos2023ss04-kubernetes-inventory-taker#28]
## User story
1. As a developer
2. I want / need a CICD pipeline
3. So I can make sure pull requests is buildable and tested

## Acceptance criteria
- [x] Proxy is builded successfully
- [x] Proxy unit tests are successfully
- [x] Frontend is builded successfully

## Linked Issues
- **Blocked by:**
- **Related to:**
- **Implements**

## Additional information
- PowerPoint draft is also fine

## Definition of done (DoD)
* Colours match with [color scheme](https://amos-p4.atlassian.net/l/cp/g5VzjL88)

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
